lib/transmuxer.js

```diff
@@ -1029,61 +1142,27 @@ Transmuxer = function() {
   this.metadataStream = new muxjs.mp2t.MetadataStream();
 
   // assemble MPEG2-TS packets into elementary streams
-  packetStream.pipe(parseStream);
-  parseStream.pipe(elementaryStream);
+  packetStream
+    .pipe(parseStream)
+    .pipe(elementaryStream);
+
   // remux the streams
   elementaryStream.pipe(aacStream);
   elementaryStream.pipe(h264Stream);
   elementaryStream.pipe(this.metadataStream);
 
-  // handle incoming data events
-  h264Stream.on('data', function(data) {
```

Comment:
This code has moved into the `push` function for the `VideoStream` constructor above. I saw no good reason for it to be here cluttering up the `Transmuxer` code.